### PR TITLE
Snake case convert

### PR DESCRIPTION
**Pull request by HawaiianSpork**

### CODE REVIEW

**Comment by HawaiianSpork** [comment]
@jrvidal would you review this pull request?

File: tests/utilities.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var assert = require('assert');
+var rewire = require('rewire');
```
Comment (jrvidal) [inline tests/utilities.js:4]: Too bad that we need this wizardry to write our unit tests. But if they're OK with it upstream, I won't complain.

File: tests/utilities.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            'should leave camelCase as camelCase': function(camelCase) {
+                assert.equal(camelCase('alreadyCamel'), 'alreadyCamel');
+            },
+            'should convert kebab-case to camelCase': function(camelCase) {
+                assert.equal(camelCase('kebab-case'), 'kebabCase');
+                assert.equal(camelCase('double-kebab-case'), 'doubleKebabCase');
+            },
+            'should not convert snake_case to camelCase': function(camelCase) {
+                assert.equal(camelCase('snake_case'), 'snake_case');
+                assert.equal(camelCase('double_snake_case'), 'double_snake_case');
```
Comment (jrvidal) [inline tests/utilities.js:28]: I feel we need a test for an identifier mixing both snake_case and kebab-case.

**Comment by jrvidal** [comment]
Back to you @HawaiianSpork

**Comment by HawaiianSpork** [comment]
@jrvidal I added the test for mixing snake and kebab case.

**Comment by jrvidal** [comment]
LGTM

**Pull request merged**